maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/MavenProjectBuilder.java

```diff
@@ -257,6 +263,43 @@ else if (pluginName != null) {
 	}
 
 	public IStatus initBundle(IProject project, String bundleUrl, IProgressMonitor monitor) throws CoreException {
+		if (bundleUrl != null) {
+			File pomFile = FileUtil.getFile(project.getFile("pom.xml"));
+
+			MavenXpp3Reader mavenReader = new MavenXpp3Reader();
+			MavenXpp3Writer mavenWriter = new MavenXpp3Writer();
+
+			try (FileReader reader = new FileReader(pomFile)) {
+				Model model = mavenReader.read(reader);
+
+				try (FileWriter fileWriter = new FileWriter(pomFile)) {
+					if (model != null) {
+						List<Plugin> plugins = model.getBuild().getPlugins();
+
+						for (Plugin plugin : plugins) {
+							if (plugin.getArtifactId().equals("com.liferay.portal.tools.bundle.support")) {
+								Xpp3Dom configuration = new Xpp3Dom("configuration");
+								Xpp3Dom url = new Xpp3Dom("url");
+
+								url.setValue(bundleUrl);
+
+								configuration.addChild(url);
+
+								plugin.setConfiguration(configuration);
+
+								break;
+							}
+						}
+
+						mavenWriter.write(fileWriter, model);
```

Comment:
Only write if you actually found the configuration to set.

Also what if there is already a configuration value?